src/components/CustomButtons.tsx

```diff
@@ -7,7 +7,7 @@ type UploadButtonProps = {
 };
 
 export const UploadButton = ({ text, onClick, isLoading }: UploadButtonProps): JSX.Element => (
-	<Button variant="inline" borderRadius="lg" onClick={onClick} isLoading={isLoading} id="ipc-upload-button">
+	<Button variant="inline" w="80%" borderRadius="lg" onClick={onClick} isLoading={isLoading} id="ipc-upload-button">
```

Comment:
I'm not a fan of this width of buttons, it's a bit to big, maybe a `w="70%"` will be better.

Reply:
> I'm not a fan of this width of buttons, it's a bit to big, maybe a `w="70%"` will be better.

I'm not a fan on ``w="70%"`` because of the ``Deploy a program`` button. The goal was to have the same width of button on both as I didn't think it would be good if the two buttons weren't the same size.
![image](https://user-images.githubusercontent.com/47077683/173129513-759111ec-ea88-4be8-91ce-eda2a5509ffa.png)

Reply:
It's really weird, because I have this for `w="70%"`:
![Screenshot from 2022-06-10 20-48-23](https://user-images.githubusercontent.com/72020723/173131110-d073a356-9df9-4b8e-b34c-f6933ffdf0e9.png)

But I agree with you on your screenshot, it's really bad.
The problem that I saw was the button with the `width="80%"` was very too large, the solution is to have a button with the same width as the tabs.